pkgs/servers/sql/mariadb/cmake-libmariadb-includedir.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/libmariadb/mariadb_config/libmariadb.pc.in b/libmariadb/mariadb_config/libmariadb.pc.in
+index b7d1f18c..b52c9b94 100644
+--- a/libmariadb/mariadb_config/libmariadb.pc.in
++++ b/libmariadb/mariadb_config/libmariadb.pc.in
+@@ -6,7 +6,7 @@
+ #  Dan Nicholson’s Guide to pkg-config (http://www.freedesktop.org/wiki/Software/pkg-config/)
+ #
+ 
+-prefix=@CMAKE_INSTALL_PREFIX@
++prefix=@CMAKE_INSTALL_PREFIX_DEV@
+ includedir=${prefix}/@INSTALL_INCLUDEDIR@/@SUFFIX_INSTALL_DIR@
+ libdir=${prefix}/@INSTALL_LIBDIR@/@SUFFIX_INSTALL_DIR@
```

Comment:
Shouldn't this one be in "out"?

Reply:
These files are used to build other packages. Checked by file structure in mariad connector-c.
Packages mariadb-server-dev = mariadb-client-dev = mariadb-connector-c